website/snippets/_packages_or_dependencies.md

```diff
@@ -0,0 +1,32 @@
+
+## Use cases
+
+Starting from dbt v1.6, `dependencies.yml` has replaced `packages.yml`. The `dependencies.yml` file can now contain both types of dependencies: "package" and "project" dependencies.
+- "Package" dependencies lets you add source code from someone else's dbt project into your own, like a library. 
+- "Project" dependencies provide a different way to build on top of someone else's work in dbt. Refer to [Project dependencies](/docs/collaborate/govern/project-dependencies) for more info.
+- You can rename your existing `packages.yml` to `dependencies.yml` unless you need to use Jinja within your packages specification. This is necessary if you want to add an environment variable with a Git token in a private Git package specification.
+
+There are some important differences between using a `dependencies.yml` compared to a `packages.yml` file:
+
+<Tabs>
+<TabItem value="dependencies" label="When to use dependencies.yml">
+
+`dependencies.yml` is designed for the [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) and cross-project reference workflow. Consider using it in the following scenarios:
+
+- Use `dependencies.yml` when you need to set up cross-project references between different dbt projects, especially in a dbt Mesh setup.
+- Use `dependencies.yml` when you want to include both projects and non-private dbt packages in your project's dependencies. 
+  - Private packages are not supported in `dependencies.yml` because they intentionally don't support Jinja rendering or conditional configuration. This is to maintain static and predictable configuration and ensures compatibility with other services, like dbt Cloud.
+- Use `dependencies.yml` for organization and maintainability. It can help maintain your project's organization by allowing you to specify [dbt Hub packages](https://hub.getdbt.com/) like `dbt_utils`. This reduces the need for multiple YAML files to manage dependencies.
+
+</TabItem>
+
+<TabItem value="packages" label="When to use packages.yml">
+
+Packages allow you to add source code from someone else's dbt project into your own, like a library. Consider using it in the following scenarios:
+
+- Use `packages.yml` when you want to download dbt packages, such as dbt projects, into your root or parent dbt project. Something to note is that it doesn't contribute to the dbt Mesh workflow.
+- Use `packages.yml` to include packages, including private packages, in your project's dependencies. If you have private packages that you need to reference, `packages.yml` is the way to go.
```

Comment:
Of all the information in this section, I think this is the one point that needs explaining & emphasizing.

I think we should:
- Link to [the docs](https://docs.getdbt.com/docs/build/packages#git-token-method) on embedding a secret via Jinja `{{ env_var() }}`
- Offer some justification in this vein:

> Embedding a git token via Jinja is a workaround for supporting private packages (repositories) via `git` installation. We are aware that this experience is not ideal, as it requires creating a machine user and distributing a git access token. While it remains the currently supported workflow, we plan to add another way of achieving this goal in the future, without requiring the use of a Jinja-embedded secret environment variable. For that reason, `dependencies.yml` does not support Jinja.